src/getters.js

```diff
@@ -1,13 +1,12 @@
-"use strict"
-
+const fetchImpl = typeof fetch !== 'undefined' ? fetch : require('isomorphic-fetch');
```

Comment:
we try to stay away from short forms / abbreviations - what does `fetchImpl` mean?

Reply:
fetch implementation. Don't do any browser work usually so not sure what the best practice is for having different implementations across different environments.